packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.test.js

```diff
@@ -0,0 +1,471 @@
+            firstName: 'Jane',
+            lastName: 'Smith',
+            address1: '456 Another St',
+            city: 'Another City',
+            stateCode: 'NY',
+            postalCode: '67890'
+        }
+    ]
+}
+
+const mockProducts = {
+    data: [
+        {
+            id: 'product-1',
+            name: 'Test Product 1',
+            imageGroups: [
+                {
+                    viewType: 'small',
+                    images: [{disBaseLink: 'https://test-image-1.jpg'}]
+                }
+            ],
+            variationAttributes: [
+                {
+                    id: 'color',
+                    name: 'Color',
+                    values: [{value: 'red', name: 'Red'}]
+                },
+                {
+                    id: 'size',
+                    name: 'Size',
+                    values: [{value: 'M', name: 'Medium'}]
+                }
+            ]
+        },
+        {
+            id: 'product-2',
+            name: 'Test Product 2',
+            imageGroups: [
+                {
+                    viewType: 'small',
+                    images: [{disBaseLink: 'https://test-image-2.jpg'}]
+                }
+            ],
+            variationAttributes: [
+                {
+                    id: 'color',
+                    name: 'Color',
+                    values: [{value: 'blue', name: 'Blue'}]
+                },
+                {
+                    id: 'size',
+                    name: 'Size',
+                    values: [{value: 'L', name: 'Large'}]
+                }
+            ]
+        }
+    ]
+}
+
+const defaultProps = {
+    basket: mockBasket,
+    onSubmit: jest.fn(),
+    submitButtonLabel: {
+        defaultMessage: 'Continue',
+        id: 'checkout.button.continue'
+    },
+    addNewAddressLabel: {
+        defaultMessage: 'Add New Address',
+        id: 'checkout.button.add_new_address'
+    },
+    noItemsInBasketMessage: {
+        defaultMessage: 'No items in basket.',
+        id: 'shipping_address.message.no_items_in_basket'
+    },
+    deliveryAddressLabel: {
+        defaultMessage: 'Delivery Address',
+        id: 'shipping_address.label.delivery_address'
+    }
+}
+
+const renderWithIntl = (component) => {
+    return render(
+        <CurrencyProvider currency="USD">
+            <IntlProvider locale="en">{component}</IntlProvider>
+        </CurrencyProvider>
+    )
+}
+
+describe('ShippingMultiAddress', () => {
+    beforeEach(() => {
+        useProducts.mockReturnValue({
+            data: {
+                'product-1': mockProducts.data[0],
+                'product-2': mockProducts.data[1]
+            }
+        })
+        useCurrentCustomer.mockReturnValue({
+            data: mockCustomer
+        })
+        useCurrentBasket.mockReturnValue({
+            data: mockBasket
+        })
+    })
+
+    afterEach(() => {
+        jest.clearAllMocks()
+    })
+
+    it('should render empty state when no items in basket', () => {
```

Comment:
Nit: please use `test` for consistency cross the code base
```suggestion
    test('should render empty state when no items in basket', () => {
```